src/client/components/input/partial-date.js

```diff
@@ -74,10 +97,20 @@ class PartialDate extends React.Component {
 		}
 	}
 
+	/**
+	* Calls the date validator function of the 'PartialDate' class.
+	* @returns {boolean} - The result as obtained from the date validator.
+	*/
 	valid() {
 		return PartialDate.validate(this.input.getValue().trim());
 	}
 
+	/**
+	* Checks that whether the editor has succesfully filled the date field in
```

Comment:
It does do this, but that's not the purpose of this function. It exists to provide the correct validation CSS class to the field input.